cedar-policy-validator/src/types.rs

```diff
@@ -2393,13 +2319,29 @@ mod test {
         );
     }
 
+    // Test display for types that don't roundtrip.
     #[test]
-    fn test_non_schema_type_display() {
-        assert_displays_as(Type::Never, r#"{"type":"Never"}"#);
-        assert_displays_as(Type::True, r#"{"type":"True"}"#);
-        assert_displays_as(Type::False, r#"{"type":"False"}"#);
-        assert_displays_as(Type::any_set(), r#"{"type":"Set"}"#);
-        assert_displays_as(Type::any_entity_reference(), r#"{"type":"Entity"}"#);
+    fn test_type_display() {
+        // Entity types don't roundtrip because the human format type parser
+        // checks that they are defined already, so we'd need to provide a
+        // complete schema. TODO: the final stage of schema parsing already does
+        // this. Can we remove duplicated checks from human schema parsing?
```

Comment:
Would be good to address this TODO in the PR, or file an issue for later & add the issue number to the comment

Reply:
I think it's not necessarily an easy change. #711